packages/pros/src/task/local.rs

```diff
@@ -50,7 +56,7 @@ fn fetch_storage() -> &'static RefCell<ThreadLocalStorage> {
 
     // Get the thread local storage for this task.
     // Creating it if it doesn't exist.
-    // This is safe as long as index 0 of the freeRTOS TLS is never set to any other type.
+    // SAFETY: in pros-rs, index 0 of the freeRTOS TLS is never set to any other type.
```

Comment:
I prefer the old comment. Users can still break this code because they have access to `pros-sys`. It's not true that index 0 will *never* be any other type.